libs/frontend/feature-build/src/components/BuildDndOverlay.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="build-dnd-overlay" :style="overlayStyle" />
+</template>
+
+<script lang="ts" setup>
+import { computed } from 'vue'
+import { runtimeContext } from '@pubstudio/frontend/util-runtime'
+import { useBuild } from '../lib/use-build'
+
+const hoverOutlineThickness = 6
+const outlineColor = '#000'
```

Comment:
I experimented a little with the outline colors, so far this worked best for me. I made side-hover borders little thicker, and separated the colors since it was a little confusing. I added a white border style to help indicate when the background is black. It doesn't look perfect, but seems to work.

Eventually we might be able to check the hovered component style and choose a single hover color based on that, but I don't think it's necessary now.